source/jormungandr/jormungandr/realtime_schedule/synthese.py

```diff
@@ -138,11 +141,9 @@ def _get_next_passage_for_route_point(self, route_point, count=None, from_dt=Non
             return None
 
         logging.getLogger(__name__).debug("synthese response: {}".format(r.text))
-        stop_point_id = str(route_point.fetch_stop_id(self.object_id_tag))
-        route_id = str(route_point.fetch_route_id(self.object_id_tag))
-        route_point = SyntheseRoutePoint(route_id, stop_point_id)
-        m = self._get_synthese_passages(r.content)
-        return m.get(route_point)# if there is nothing from synthese, we keep the base
+        passages = self._get_synthese_passages(r.content)
+
+        return self._find_route_point(route_point, passages)
```

Comment:
don't really like the name of this function, I would expect a route point as result while it will return passages